app/resources/v1/streets.js

```diff
@@ -237,12 +235,12 @@ exports.get = function (req, res) {
 } // END function - exports.get
 
 exports.find = function (req, res) {
-  var creatorId = req.query.creatorId
-  var namespacedId = req.query.namespacedId
-  var start = (req.query.start && parseInt(req.query.start, 10)) || 0
-  var count = (req.query.count && parseInt(req.query.count, 10)) || 20
+  let creatorId = req.query.creatorId
```

Comment:
Go through this function and make sure all the variables defined as `let` can actually be `const`